update_gas_prices.py

```diff
@@ -64,26 +38,26 @@
     # Prepare the dictionaire
     dict_prices = {
         PREVIOUS_WEEK: current_data[CURRENT_WEEK],
-        CURRENT_WEEK: {START_DATE_KEY: start_date, END_DATE_KEY: end_date, GAS_KEY: {}},
+        CURRENT_WEEK: {START_DATE_KEY: start_date, END_DATE_KEY: end_date, GAS_KEY: gas_info},
     }
+
     # Parse the data
-    i += 1
-    while i < len(gas_info) - 1:
-        gas = gas_info[i]
+    for key, value in gas_info.items():
         price = None
         # skip to next line in case fuelPrice is not a float
         while price is None:
             try:
-                price = float(gas_info[i + 1][:-1].replace(",", "."))
+                price = float(value.replace(",", "."))
             except ValueError:
-                i += 1
+                continue
+
+        dict_prices[CURRENT_WEEK][GAS_KEY][key] = price
 
-        dict_prices[CURRENT_WEEK][GAS_KEY][gas] = price
-        i += 2
     # Add Gasoline 98 price
     dict_prices[CURRENT_WEEK][GAS_KEY][GASOLINE_98] = round(
         dict_prices[CURRENT_WEEK][GAS_KEY][GASOLINE_95] + DIFFERENCE_95_98_PRICE, 3
     )
+    
     # Make tweet
     make_tweet(dict_prices)
```

Comment:
Didn't check the tweet part, but it seems good since you have the update here:

<blockquote class="twitter-tweet"><p lang="es" dir="ltr">— Devo abastecer? ⛽️ <br><br> 2023-10-09 | 2023-10-02<br> a | a<br> 2023-10-15 | 2023-10-08<br><br>Gasóleo 1.478€ ⬇️️ 1.509€<br>Gasolina 95 1.667€ ⬇️️ 1.711€<br>Gasolina 98 1.817€ ⬆️ 1.8€</p>&mdash; Devo Abastecer (@devoabastecer) <a href="https://twitter.com/devoabastecer/status/1713256358651318588?ref_src=twsrc%5Etfw">October 14, 2023</a></blockquote>

Reply:
Yes, it should be okay, we didn't change anything